packages/mdc-switch/_variables.scss

```diff
@@ -0,0 +1,32 @@
+// 
+// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+$mdc-switch-track-width: 34px;
+$mdc-switch-track-height: 14px;
+$mdc-switch-knob-diameter: 20px;
+$mdc-switch-focus-ring-diameter: 48px;
+$mdc-switch-knob-active-margin: $mdc-switch-track-width - $mdc-switch-knob-diameter;
+
+$mdc-switch-unchecked-background-color: #221f1f;
+$mdc-switch-unchecked-knob-color: #f1f1f1;
+$mdc-switch-unchecked-focus-ring-color: #9e9e9e;
+$mdc-switch-disabled-focus-ring-color: #000;
+$mdc-switch-unchecked-disabled-knob-color: #bdbdbd;
```

Comment:
unchecked track color and disabled unchecked track color are the same value with different opacities, so I just made it `mdc-switch-unchecked-track-color`